### PR TITLE
PP-2809 Add Direct Debit Frontend Npm express app structure

### PR DESCRIPTION
**Pull request by rauligs**
## WHAT   
 Add base Npm Express application with healthcheck resource:
- Docker, Snyk, Lint support
- Routing
- Healthcheck
- Include gov.uk templates support
- I18n support
- Public assets processing (css, js, images)

### CODE REVIEW

**Comment by jonheslop** [comment]
I know @brid has done a load of work to make the express app and template rendering better would be good to get that stuff into this

If you guys are interested in trying out Webpack, I'd be happy to fork a branch and have a go at putting it all together?

**Comment by DanailMinchev** [comment]
Thanks @jonheslop . Yes, Webpack sounds great!

File: app/utils/no_cache.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-module.exports = function (res) {
+module.exports = (res) => {
```
Comment (jonheslop) [inline app/utils/no_cache.js]: Small thing but if you only have one variable you don't need the brackets

So can be `module.exports = res => {...}`

File: app/router.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const { healthcheck } = paths
 
 module.exports.paths = paths
 
-module.exports.bind = function (app) {
+module.exports.bind = (app) => {
```
Comment (jonheslop) [inline app/router.js]: Small thing but if you only have one variable you don't need the brackets

So can be module.exports.bind = app => {...}

Comment (rauligs) [reply app/router.js]: Done

File: app/utils/no_cache.js (hunk 1 above)
Comment (rauligs) [reply app/utils/no_cache.js]: Done

**Comment by rauligs** [comment]
Removed _winston_ dependency. Took notes about bringing logging with _pino_

**Pull request merged**